scripts/o3de/o3de/register.py

```diff
@@ -614,6 +614,13 @@ def register_repo(json_data: dict,
 
     return 1
 
+def register_remote_repo(json_data: dict,
+                  repo_uri: str,) -> int:
+    # get the o3de_manifest.json file and insert new remote repo in the repos field
+    json_data.setdefault('repos', []).insert(0, repo_uri)
+    # save updated o3de_manifest.json file
+    manifest.save_o3de_manifest(json_data)
+    return 0
```

Comment:
Did the existing `register(repo_uri=)` function not work?
```suggestion
```

Reply:
Yeah, the register function would take in the repo_uri as argument that then calls the register_repo function that would include parsing the uri and downloading from the provided uri.

Reply:
If I use the register.register function, the repo_uri would call the register_repo function in register.py that would require downloading of the repo_uri provided. Should I try to make change to to register_repo function instead?